index.js

```diff
@@ -112,3 +168,7 @@ function padStart (what, target, start) {
 function padEnd (what, target, start) {
   return what + repeat(' ', target - what.length)
 }
+
+function max (a, b) {
```

Comment:
Why use this instead of `Math.max`?

Reply:
Originally because I used it in a `reduce` (where the "only two arguments" is important). I just kept it for the looks.
I'm fine with dropping it.